### ISSUE TITLE
Scale data augmentation in object detection YOLOv11

### ISSUE THREAD

**Issue by UranLajciSolaborate** [ultralytics/ultralytics#17767]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

If I do not want the scale data augmentation should I set it to 1 or 0.

What we get from the docs:
 	float 	0.5 	>=0.0 	Scales the image by a gain factor, simulating objects at different distances from the camera.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @UranLajciSolaborate, thank you for reaching out to us at Ultralytics 🚀!

For questions regarding data augmentation or other custom training queries, please ensure you're providing as much context as possible, including any specific configurations or code snippets you are using. This helps us assist you more effectively.

If you're encountering a 🐛 Bug, kindly provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). This will enable us to better diagnose and tackle any issues you might be facing.

In the meantime, I recommend checking our [Docs](https://docs.ultralytics.com) for detailed explanations and examples on usage, which often address common questions. You can also refer to our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to optimize your custom training setups.

We also have a vibrant community eager to help. For real-time discussions, feel free to join us on [Discord](https://ultralytics.com/discord) 🎧, or engage with extended threads on our [Discourse](https://community.ultralytics.com) platform. You can also join our [Subreddit](https://reddit.com/r/ultralytics) to share insights and knowledge.

## Upgrade

Don't forget to keep your environment updated! You can upgrade to the latest `ultralytics` package along with all necessary [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) within a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) by running the following command:

```bash
pip install -U ultralytics
```

## Environments

Remember, you can seamlessly run YOLO in any of the following verified environments, with all dependencies pre-installed:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If the above badge is green, this means all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing, verifying the optimal operation of YOLO on various tasks and modes across different platforms.

This automated response is just the beginning—rest assured, an Ultralytics engineer will soon be here to provide further assistance. Thank you for your patience and collaboration! 😊

**Comment by Y-T-G** [comment]
0

**Comment by UranLajciSolaborate** [comment]
![image](https://github.com/user-attachments/assets/ca7b1c7c-2741-4357-a507-15180ce3a02c)
![image](https://github.com/user-attachments/assets/26257b01-ef1e-407e-b63a-9e6e73bc32c8)

What we understand from the Ultralitics code is that setting scale to 0 it will disable the scale augmentation.

**Comment by glenn-jocher** [comment]
@UranLajciSolaborate yes, that is correct. Setting `scale` to `0` in the Ultralytics configuration will disable scale augmentation.

**Comment by glenn-jocher** [comment]
You're absolutely right, @vittorio-prodomo. Thank you for this valuable feedback. The current documentation could indeed be clearer about how the `scale` parameter works. 

As you pointed out, the behavior isn't simply multiplying by the scale factor. Instead, when `scale=0.5`, the actual scaling factor is randomly chosen between `1-scale` (0.5) and `1+scale` (1.5). That's why `scale=0` disables augmentation (no scaling variation), while `scale=1.0` would actually create quite extreme random scaling between 0 and 2.0.

We'll work on improving this in the documentation to make it more explicit. Thanks for helping make our docs better for all users!
